### PR TITLE
fix: User avatar shrinks on mobile when action is submitting

### LINKED ISSUE #2546: User avatar shrinks on mobile when action is submitting
## Steps to reproduce

1. Set screen size to mobile
2. Create a new action and submit

https://github.com/JoinColony/colonyCDapp/assets/34915414/f3b97d44-7e0f-44b0-88a4-999748d2f1a5

## Expected behaviour

* User avatar remains at the standard size

## Actual behaviour

* User avatar shrinks

(This one has been resolved before, but seems to have come back)

### PR DESCRIPTION
**Pull request by Nortsova**
## Description
Fix user avatar shrinks on mobile when action is submitting

## Testing
* Step 1. Set screen size to mobile
* Step 2. Create a new action and submit
* Step 3. See that user avatar is visible
image

Resolves #2546

### CODE REVIEW

**Review by davecreaser — DISMISSED** [review]
Good stuff, all tested and the avatar stays visible!

Screenshot 2024-06-18 at 16 25 15

File: src/components/common/Extensions/UserHubButton/UserHubButton.tsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ const UserHubButton: FC = () => {
           {
             '!border-blue-400': visible || isUserHubOpen,
           },
-          'md:hover:!border-blue-400',
+          ' flex min-w-[3rem] md:hover:!border-blue-400',
```
Comment (iamsamgibbs) [inline src/components/common/Extensions/UserHubButton/UserHubButton.tsx:154]: I'm being incredibly nitpicky, but there is an unnecessary space at the start of this line.

**Review by rumzledz — APPROVED** [review]
Looking brill @Nortsova ✨

Screenshot 2024-06-18 at 17 01 55

File: src/components/common/Extensions/UserHubButton/UserHubButton.tsx (hunk 1 above)
Comment (iamsamgibbs) [reply src/components/common/Extensions/UserHubButton/UserHubButton.tsx:154]: I also don't think flex is needed here as it is already included in the button component.

File: src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx [hunk 2]
```diff
@@ -117,7 +117,7 @@ const NavigationSidebarContent: FC<NavigationSidebarProps> = ({
             md:gap-9
           `}
         >
-          <div className="flex gap-6 md:flex-col md:gap-9">
+          <div className="flex gap-3 md:flex-col md:gap-9">
```
Comment (iamsamgibbs) [inline src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx:120]: This change affects a different part of the header.

**Review by iamsamgibbs — CHANGES_REQUESTED** [review]
Congratulations on your first PR @Nortsova 🎉 

The fix does indeed solve the issue:

https://github.com/JoinColony/colonyCDapp/assets/34915414/80f47a09-6b9c-4060-ad0a-75c76ceb772c

However, I think this change is unnecessary and is affecting a different part of the header:

Screenshot 2024-06-18 at 17 13 11

And `flex` is already included in the button component and doesn't need to be included again here:
 
Screenshot 2024-06-18 at 17 11 47

Nice job identifying the components involved and making the fix though! I'll review again after these changes.

**Review by iamsamgibbs — APPROVED** [review]
Looks good to me now! Thanks for making the changes :)

Screenshot 2024-06-19 at 10 15 46

**Review by rumzledz — APPROVED** [review]

**Review by mmioana — APPROVED** [review]
Avatar no longer shrinks

Screenshot 2024-06-21 at 10 25 23


Great work @Nortsova! 🥳

**Pull request merged**